### DISCUSSION TITLE
【招聘】【全职贡献者：后端】【远程办公】Go语言｜2500 u/月

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#777] [General]
由于 WTF Academy 发展需求，现招一个全职后端贡献者（远程办公，Go语言），主要任务是开发 wtf.academy 教育平台后端，欢迎大家推荐！

## 1. 任职要求
1. 两年以上的 Golang 后端开发经验，熟悉 Golang 常见开发框架，如 Gin、Echo 等，熟悉 Golang 并发编程和 GC 机制。
2. 至少熟悉一种关系型数据库，如 MySQL、PostgreSQL 等，熟悉数据库的设计和优化。
3. 熟悉缓存数据库，如 Redis、Memcached，熟悉常见的消息队列系统，如 RabbitMQ、Kafka 等。
4. 熟悉 HTTP、TCP/IP 等网络协议，了解 RESTful API 设计规范。
5. 了解 Linux 操作系统，熟悉常用的 Shell 指令，熟悉 Git 版本控制。
6. 了解容器化技术（如 Docker）、持续集成/持续部署（CI/CD）流程，以及云服务平台（如AWS、GCP）的使用。
7. 熟练编写单元测试和集成测试，确保核心业务流程的正确性和稳定性。

加分项：
- WTF 贡献者
- 了解 Solidity，完成 WTF Solidity 101 课程
- 有开源贡献经历
- 熟悉 Node.js 和 Rust
- 熟悉英文

## 2. 我们对你的期望
1. 适应远程办公，在不面对面的情况下也能很好的与其他贡献者沟通
2. 自觉并出色的完成任务，每月大约 20 工作日（每日 8 小时）的工作量
3. 灵活安排工作时间，适应下午和晚上工作，周会一般安排在周末晚上（兼职贡献者一般周末才有时间）

## 3. 赏金
1. 基础赏金：2500 u/月（3个月试用期）
3. 完成项目或大版本更新会发项目赏金，但会兼顾公平，向兼职贡献者倾斜
5. 优先获得其他 Web3 项目给 WTF Academy 的空投机会

## 4. 报名链接

https://forms.gle/ititfZv18pVCjNYd9

**Comment by XiaoK29** [comment]
已投，混个脸熟

**Comment by JDChi** [comment]
后端会有兼职的吗？

**Comment by power-cake** [comment]
有其他后端语言兼职吗？例如java，😀

**Comment by stern419** [comment]
你好，现在还招吗？

**Reply by zongh646-star** [reply]
> 你好，现在还招吗？

会渗透的话可以联系我，空闲的时候做一做，
